packages/modal-button/src/types.tsx

```diff
@@ -0,0 +1,27 @@
+import { LidoComponentProps } from '@lidofinance/utils'
+export type { Theme } from '@lidofinance/theme'
+
+export enum ModalButtonSize {
+  xxs,
+  xs,
+  sm,
+  md,
+  lg,
+}
+export type ModalButtonSizes = keyof typeof ModalButtonSize
+
+export type ModalButtonProps = LidoComponentProps<
+  'button',
+  {
+    size?: ModalButtonSizes
+    activate?: boolean
```

Comment:
Can we change the prop name to something like `active`? `activate` is a verb and doesn't semantically fit for boolean values, imo.

Reply:
Of course, I mistakenly used the verb :(